config/jobs/kubernetes-csi/csi-driver-windows-poc/csi-driver-windows-poc-config.yaml

```diff
@@ -0,0 +1,47 @@
+presubmits:
+  kubernetes-csi/csi-driver-windows-poc:
+  - name: pull-kubernetes-csi-csi-driver-windows-poc
+    always_run: false
+    decorate: true
+    extra_refs:
+    - org: jingxu97
+      repo: csi-driver-host-path
+      base_ref: windows-poc
+      path_alias: kubernetes-csi/csi-driver-host-path
+    skip_report: false
+    skip_branches: []
+    labels:
+      preset-service-account: "true"
+      preset-common-gce-windows: "true"
+      preset-e2e-gce-windows: "true"
+    annotations:
+      testgrid-dashboards: sig-storage-csi-other
+      testgrid-tab-name: pull-kubernetes-csi-csi-driver-windows-poc
+      description: Kubernetes-CSI pull job in repo csi-driver-windows-poc
+    spec:
+      containers:
+      # We need this image because it has Docker in Docker and go.
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200507-2ed1a50-master
+        command:
+        - runner.sh
+        args:
+        - bash
+        - -c
+        - >-
+          cd $(GOPATH)/src/k8s.io/kubernetes &&
```

Comment:
if you're going to CD to kubernetes in gopath you need to clone it using extra_refs

Reply:
or just skip this and use --extract=ci/latest

Reply:
I am trying to run kubetest to start a windows cluster, so I assume I need to CD to kubernetes  or just use --extract=ci/latest without CD?